packages/cojson/src/sync.ts

```diff
@@ -659,57 +667,8 @@ export class SyncManager {
         continue;
       }
 
-      // If we'll be performing transaction verification, ensure the account is available.
-      if (!this.skipVerify) {
-        const accountId = accountOrAgentIDfromSessionID(sessionID);
-
-        if (isAccountID(accountId)) {
-          const account = this.local.getCoValue(accountId);
-
-          // We can't verify the transaction without the account, so we delay the session content handling until the account is available
```

Comment:
Delaying a single session could lead to inconsistent statuses on the client, where some of the transactions could be missing.

This can be quite a problem if there are some migrations that check the presence of a property to set default values.